fontir/src/variations.rs

```diff
@@ -21,6 +21,44 @@ use write_fonts::{
 
 use crate::error::VariationModelError;
 
+/// Trait for performing banker's rounding on a value.
+///
+/// Banker's rounding rounds half to even. This means that half-way values
+/// are rounded to the nearest even number. For example, 2.5 rounds to 2.0,
+/// 3.5 rounds to 4.0, and -2.5 rounds to -2.0.
+///
+/// This is the same rounding mode as [`f64::round_ties_even`], which was
+/// stabilized in Rust 1.77.0. It also matches Python's buit-in `round` function.
+/// It is usually preferred over alternative approaches because it reduces bias
+/// and errors in calculations.
```

Comment:
Suggest elaborating on when this makes sense and ideally adding a citation. Perhaps we can reference the python PR or other artifacts where it was adopted?

Reply:
I'll add link to the wikipedia entry on Rounding and a link to the fonttools PR